Kuinox.NupkgDeterministicator.Task/MakeNukpkgDeterministic.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Text;
+
+namespace Kuinox.NupkgDeterministicator.Task
+{
+    public sealed class MakeNukpkgDeterministic : ToolTask
+    {
+        /// <summary>
+        /// The zip to make deterministic.
+        /// </summary>
+        public string ZipPath { get; set; }
+
+        /// <summary>
+        /// The last write time to sets. Default to 2000-01-01.
+        /// </summary>
+        public DateTime LastWriteTime { get; set; } = new DateTime(2000, 1, 1);
```

Comment:
I don't know which PR gets in first, but this creates a semantic merge conflict between them. SOURCE_DATE_EPOCH should clearly be used in the initializer here too.

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.datetime.unixepoch?view=net6.0
For later reference, UnixEpoch is pre-defined here.